### ISSUE TITLE
Link to nightly builds not working

### ISSUE THREAD

**Issue by Daedalus3** [nextcloud/calendar#2756]
Link included in the README.md to nightly builds returns a 404 [https://nightly.portknox.net/calendar/?C=M;O=D](url)

**Comment by tcitworld** [comment]
Yeah, this section should be removed until we have https://github.com/nextcloud/calendar/issues/129
